### PR TITLE
[styles] add amenity=brothel and amenity=stripclub icon

### PR DESCRIPTION
**Pull request by dvdmrtnz**
Add amenity=brothel and amenity=stripclub icon

https://github.com/organicmaps/organicmaps/issues/210

### CODE REVIEW

**Comment by biodranik** [comment]
Will it require changes in any age-related restrictions in app stores?

**Comment by dvdmrtnz** [comment]
> Will it require changes in any age-related restrictions in app stores?

I don't know, but the "brothel" category and POI is already present in the current AppStore version of the app but without an icon.

**Comment by RedAuburn** [comment]
google maps includes them and is rated PEGI 3, so gplay is probably fine

**Review by vng — APPROVED** [review]

**Review by strump — APPROVED** [review]

File: data/styles/clear/include/Icons.mapcss [hunk 1]
```diff
@@ -1422,6 +1422,12 @@ node|z16-[amenity=nightclub],
 area|z16-[amenity=nightclub]
 {icon-image: nightclub-m.svg;font-size: 11;}
 
+node|z16-[amenity=stripclub],
+area|z16-[amenity=stripclub],
+node|z16-[amenity=brothel],
+area|z16-[amenity=brothel]
```
Comment (pastk) [inline data/styles/clear/include/Icons.mapcss:1428]: I guess make them z17- or z18- even.
For reference, amenity-cinema is z17-, amenity-car_rental is z18-.

**Review by pastk — CHANGES_REQUESTED** [review]
Please add stripclub and brothel into `data/styles/clear/include/priorities_4_overlays.prio.txt`

**Comment by dvdmrtnz** [comment]
> Please add stripclub and brothel into `data/styles/clear/include/priorities_4_overlays.prio.txt`

Added to priorities file and moved down to z18-

**Review by pastk — APPROVED** [review]
Merge after the current release is out, as it needs a maps regen.

**Comment by d4f5409d** [comment]
Someone's in a mood 😏

**Review by jimcarst — APPROVED** [review]

**Pull request merged**